crates/timers/tests/web.rs

```diff
@@ -44,72 +45,62 @@ fn timeout_cancel() -> impl Future<Item = (), Error = wasm_bindgen::JsValue> {
     })
     .forget();
 
-    receiver.map_err(|e| e.to_string().into())
+    receiver.await.unwrap();
 }
 
-#[wasm_bindgen_test(async)]
-fn timeout_future() -> impl Future<Item = (), Error = wasm_bindgen::JsValue> {
-    TimeoutFuture::new(1).map_err(|_| "impossible".into())
+#[wasm_bindgen_test]
+async fn timeout_future() {
+    TimeoutFuture::new(1).await;
 }
 
-#[wasm_bindgen_test(async)]
-fn timeout_future_cancel() -> impl Future<Item = (), Error = wasm_bindgen::JsValue> {
+#[wasm_bindgen_test]
+async fn timeout_future_cancel() {
     let cell = Rc::new(Cell::new(false));
 
     let a = TimeoutFuture::new(1).map({
         let cell = cell.clone();
         move |_| {
             assert_eq!(cell.get(), false);
-            1
+            1 as u32
```

Comment:
Why does this need a cast? Assuming the type inference is failing, it can use `1u32` instead.